js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties

```
@@ -98,7 +98,7 @@ passwordPolicyHintsEnabledHelp=Aplicable solo para MSAD editable. Si está activ
 expirationValueNotValid=El valor debe ser mayor o igual a 1
 eventTypes.UPDATE_CONSENT.name=Actualizar consentimiento
 forceArtifactBinding=Forzar la vinculación de artefactos
-eventTypes.REFRESH_TOKEN_ERROR.description=Error de token de actualizaci�n
+eventTypes.REFRESH_TOKEN_ERROR.description=Error de token de actualizaci�n
```

Same here, only use valid UTF-8.
```suggestion
eventTypes.REFRESH_TOKEN_ERROR.description=Error de token de actualización
```

---

Done, I apologise as i didn't see it

---

No problem, you already removed a bunch, which is great!